assets/src/domain/eventEditor/ui/datetimes/dateForm/useDateFormConfig.ts

```diff
@@ -39,10 +38,10 @@ const useDateFormConfig = (id: EntityId, config?: EspressoFormProps): DateFormCo
 	const initialValues: DateFormShape = {
 		...pick<Partial<Datetime>, keyof Datetime>(FIELD_NAMES, restProps),
 		dateTime: {
-			startDate: format(startDate, CONVERT_TO_MOMENT_DATE_FORMAT),
-			startTime: format(startDate, CONVERT_TO_MOMENT_TIME_FORMAT),
-			endDate: format(endDate, CONVERT_TO_MOMENT_DATE_FORMAT),
-			endTime: format(endDate, CONVERT_TO_MOMENT_TIME_FORMAT),
+			startDate: startDate,
+			startTime: startDate,
+			endDate: endDate,
+			endTime: endDate,
 		},
```

Comment:
imho this is the kind of thing (the format/shape that dates are supplied as) that we should be trying to encapsulate within our infrastructure adapters because if we ever have to change datepickers in the future, then this code will also need to change. Ideally our adapters would always receive the exact same input (JS Date object, ISO date string, etc) and then convert to whatever the third party code expects (JS Date object, moment object, ISO date string, etc).

Reply:
good one, agree, the end components and the application layer shouldn't be worried too much about these formatting